File: SchoolRegisterApp.Models/Dtos/PersonHistoryDto.cs
using SchoolRegisterApp.Models.Enums;

namespace SchoolRegisterApp.Models.Dtos
{
    public class PersonHistoryDto
    {
        public int Id { get; set; }

        public int PersonId { get; set; }

        public int UserId { get; set; }

        public DateTime ActionDate { get; set; }

        public DataModified DataModified { get; set; }

        public ModificationType ModificationType { get; set; }
    }
}

File: school-register-app/src/person-history/dtos/person-history.dto.ts
import { DataModifiedEnum } from "../../enums/data-modified.enum";
import { ModificationTypeEnum } from "../../enums/modification-type.enum";

export class PersonHistoryDto { 
  id: number;
  personId: number;
  userId: number;
  actionDate: Date;
  dataModified: DataModifiedEnum;

  modificationType: ModificationTypeEnum;
}
